python/vyos/util.py

```diff
@@ -35,7 +35,10 @@ def debug(flag):
     # this is to force all new flags to be registered here to be documented:
     if flag not in ['developer', 'ifconfig']:
         return ''
-    return flag if os.path.isfile(f'/tmp/vyos.{flag}.debug') else ''
+    for folder in ('/tmp', '/config'):
```

Comment:
Is there any documentation in readthedocs or the source about the new location?

Reply:
How can there be documentation for a feature which does not exist yet? 😛 

Also, the only reason I know about this feature is as I read the code and "stubbled" upon it.

I was not aware it was supposed to be documented as searching for it does not return anything:

```
https://docs.vyos.io/en/latest/search.html?q=vyos.debug.ifconfig
<no result>
```

on the latest master of vyos-documentation:
```
# grep "vyos.ifconfig.debug" . -r
#
```

Is it supposed to be documented ?

Reply:
The old approach with the /tmp file was documented in the source, so should be the new one. As it is now persistend it should also be mentioned here: https://docs.vyos.io/en/latest/contributing/development.html#debugging but his can happen after this is merged.

Reply:
Happy to update the documentation, but as I failed to noticed it was public, I should also add a "command" debugging as we are at it.

Reply:
Everything which is not yet documented should be added into the debugging section for all